worlds/sc2/Options.py

```diff
@@ -655,7 +694,12 @@ def verify(self, world: Type['World'], player_name: str, plando_options: PlandoO
         self.value = new_value
         for item_name in self.value:
             if item_name not in world.item_names:
-                picks = get_fuzzy_results(item_name, list(world.item_names), limit=1)
+                from . import ItemGroups
```

Comment:
Alas, trying to import this in the top of the file results in a circular dependency and the generator always crashing. Another option would be to put the mission group names on the `world` object, though that would be tightening the assumptions on when this function is called, and the PR on main makes me think that interface is not entirely nailed down yet.